### ISSUE TITLE
Baseline CarRacing GAIL

### ISSUE THREAD

**Issue by MHong94** [rohitrango/BC-regularized-GAIL#2]
I'm trying to run the simple baseline GAIL for CarRacing Env.  In order to record and save the expert trajectory, it requires a trained agent from RL.  I am trying to load in agents from other repositories such as https://github.com/xtma/pytorch_car_caring, but I am failing to do so.  Is the trained agent you used available? (Or even simpler, the expert_data.pkl for CarRacing)

**Comment by rohitrango** [comment]
Hi,

Here is the data that I have. It contains a "pkl" file which you need to open with torch (sorry for the incorrect extension). The file contains a dictionary with list of state, action, start of episode (to keep track where one episode ends), and reward. The state is a path to the image, which can be loaded using `np.load`.

Here is a code snippet to get started (after unzipping):
```
>>> import torch
>>> data  = torch.load("expert_data.pkl")
>>> data['obs']
[path/to/1, /path/to/2 ....]
>>> data['reward']
[rew0, rew1 ...]

```

Here is the link: https://drive.google.com/file/d/1LOeYiQWn3sfw31uwKq5aFF_xEyQv5Lms/view?usp=sharing

Let me know if you need any help.

**Comment by MHong94** [comment]
Do you have a shell script that runs gail for CarRacing?  I was able to get it running, but I had to make a several changes and the rewards don't seem to be increasing.

**Comment by rohitrango** [comment]
Here's the command that I used:

```
python main.py --env-name CarRacing-v0 --algo ppo --use-gae --lr 2.5e-4 --clip-param 0.1 --value-loss-coef 0.5 --num-processes 8 --num-steps 128 --num-mini-batch 4 --log-interval 1 --use-linear-lr-decay --entropy-coef 0.01 --model_name CarRacingPPO
```
